src/webgpu/api/operation/texture_view/format_reinterpretation.spec.ts

```diff
@@ -0,0 +1,68 @@
+export const description = `
+Test texture views can reinterpret the format of the original texture.
+`;
```

Comment:
oh just realized this overlaps with test descriptions in the neighboring `read.spec.ts` and `write.spec.ts` e.g.:

```
g.test('format')
  .desc(
    `Views of every allowed format.
- x= every texture format
- x= every valid view format (including same)
```

Reply:
not sure if it makes more sense for these tests to be separate (in which case update the test description there) or combined with those.

Reply:
I'm keeping them separate and removed it from those files since I would think those tests are more about reading/writing to various subresources in different ways and checking the correct subresource specified by the view are written/read.

Testing the interpretation of the contents themselves feels like it should be separate.